src-django/api/views.py

```diff
@@ -80,3 +80,12 @@ class ConceptViewSet(viewsets.ModelViewSet):
 
     def get_queryset(self):
         return models.Concept.objects
+
+
+class ShowIfViewSet(viewsets.ModelViewSet):
+    model = models.ShowIf
+    serializer_class = serializer.ShowIfSerializer
+
+    def get_queryset(self):
+        user = self.request.user
+        return models.ShowIf.objects.filter(page__procedure__owner_id__exact=user.id)
```

Comment:
One thing I noticed is that we don't validate the `self.request.user` object in our viewsets so we keep getting 500 (accessing `id` of `null`) instead of 401